.idea/gradle.xml

```diff
@@ -10,6 +10,8 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/data" />
+            <option value="$PROJECT_DIR$/domain" />
```

Comment:
please, add all .idea folder into gitignore

Reply:
added folder to gitingore